dashboard/config/scripts/levels/CSPU5 Add a 2nd Screen_2018.level

```diff
@@ -55,14 +51,18 @@
     "fail_on_lint_errors": "false",
     "makerlab_enabled": "false",
     "project_template_level_name": "CSPU5 Adding Screens Template_2018",
-    "show_debug_watch": "false",
+    "show_debug_watch": "true",
     "hide_share_and_remix": "false",
     "disable_if_else_editing": "false",
     "expand_debugger": "false",
     "parent_level_id": 6741,
     "name_suffix": "_2018",
-    "contained_level_names": null
+    "preload_asset_list": null
   },
+  "game_id": 41,
+  "created_at": "2018-05-04T23:00:47.000Z",
+  "level_num": "custom",
+  "user_id": 155,
```

Comment:
Not immediately clear to me why these 4 fields got reordered, but should be semantically the same...